### PR TITLE
feat: export list of regions

### LINKED ISSUE #116: [ENHANCEMENT] Export country list using ESM import
If you want to create a page with a selector to select countries, instead of clicking on it, it's a good idea to be able to get the list of countries from the library
i took it from source code, but it should be exported
(or i missed something ?)

### PR DESCRIPTION
**Pull request by warenbe**
Hi
Added an export for the country list
each country is an object like 
```
{
  name: "name of the country",
  code: "2 letter code (US for example)"
}
```

feel free to tell me if i made a mistake as it's first time i use pull in github.

Close #116

### CODE REVIEW

**Comment by Josh-Cena** [comment]
Could you either use Yarn v1 to install, or add `yarnPath` (and commit the `yarn-3.2.0.cjs` file)? Yarn berry is not recommended to be installed globally.

File: lib/src/index.tsx [hunk 1]
```diff
@@ -179,4 +179,9 @@ export default function WorldMap(props: Props): JSX.Element {
   );
 }
 
-export { WorldMap };
+const Countries = geoData.features.map((g) => ({
```
Comment (Josh-Cena) [inline lib/src/index.tsx:182]: This should be camelCase because it's not a type or a component.

**Comment by warenbe** [comment]
well, i have an error when i use yarn v1
i fixed it with latest yarn
probably my setup.

**Comment by Josh-Cena** [comment]
What's the error?

I would revert this change before merging anyways, but still curious to know, in case you need help

**Comment by warenbe** [comment]
well i think this is a problem between github desktop and yarn
something about impossibility to find path of cygpath

**Comment by warenbe** [comment]
well, i tried and it seems i can commit with command line, but not from GH desktop.
apparently i'm not the only one to have this issue.
anyway, thank for the works and the modification ;)
i find this library very usefull: simple, clean.

**Pull request merged**